controller.go

```diff
@@ -50,8 +51,19 @@ func (c *OFController) ConnectionDown() {
 	// handle connection down
 }
 
-func (c *OFController) sendEchoLoop() {
-	// send echo request forever
+func (c *OFController) sendEchoLoop(dp *Datapath) {
+	// send echo request forever, first tick goes after
+	// echo interval
+	fmt.Println("Controller echo loop started for dp: %v", dp)
```

Comment:
can you add similar log when echo loop exits?

Reply:
sure

Reply:
Actually no point adding the log after the loop, it won't get executed as we will panic

Reply:
yeah if we panic then no point.